backend/coins/btc/account.go

```diff
@@ -99,13 +99,15 @@ type Account struct {
 	activeTxProposal     *maketx.TxProposal
 	activeTxProposalLock locker.Locker
 
-	feeTargets []*FeeTarget
+	feeTargets   []*FeeTarget
+	feeTargetsMu locker.Locker
 	// Access this only via getMinRelayFeeRate(). sat/kB.
 	minRelayFeeRate   *btcutil.Amount
 	minRelayFeeRateMu sync.Mutex
 
 	// true when initialized (Initialize() was called).
-	initialized bool
+	initialized   bool
+	initializedMu locker.Locker
```

Comment:
could this be an `atomic.bool` instead?

Reply:
No as the whole Initialize function needs to be serialized, not just writing to and reading from this variable.